libexec/rc/rc.d/motd

```diff
@@ -19,6 +19,37 @@ COMPAT_MOTD="/etc/motd"
 TARGET="/var/run/motd"
 TEMPLATE="/etc/motd.template"
 PERMS="644"
+WORLD_OPTIONS="/usr/share/misc/perf_options"
+
+append_perf_warnings()
+{
+	local _t _kernel_options _world_options _opt_type
+	_t="$1"
+
+	for _opt in DIAGNOSTIC INVARIANTS WITNESS; do
+		if check_kern_features $(echo $_opt | tr A-Z a-z); then
+			_kernel_options="${_kernel_options} $_opt"
+		fi
+	done
+	if [ -e ${WORLD_OPTIONS} ]; then
+		_world_options="$(grep -v "^#" "$WORLD_OPTIONS")"
+	fi
+	if [ -n "$_kernel_options" -o -n "$_world_options" ]; then
+		_opt_type="kernel"
+		echo >> "$_t"
+		for _opt in $_kernel_options WORLD $_world_options; do
+			case $_opt in
+			WORLD)
+				_opt_type="world"
+				;;
+			*)
+				echo -n " ${_opt}"
+				echo "WARNING: $_opt $_opt_type option defined, expect reduced performance" \ >> $_t
```

Comment:
```suggestion
				echo "WARNING: $_opt $_opt_type option defined, expect reduced performance" >> $_t
```
otherwise you get two trailing spaces (which, other than the final line which I guess you cut short, can be seen in the PR description); one as a separator, one for the argument

Reply:
Oops, leftover from me decided I didn't care about line length and unwrapping the lines.